modules/lazyload-vimeo/index.js

```diff
@@ -38,6 +38,10 @@
      */
     $(document).ready(doload_llv()).ajaxStop(function () {
       doload_llv();
+      if (typeof responsiveVideos.resize === 'function' && $_o.responsive === true ) {
```

Comment:
Awesome @devattendant, thanks for figuring this out and contributing!
Before I merge this, one suggestion/ask: 

As you know, the file already includes this:
```
    if (typeof responsiveVideos.init === 'function' && $_o.responsive === true) {
      responsiveVideos.init();
    } else {
      markInitialized();
    }

    if (typeof $_o.callback === 'function') {
      $_o.callback();
    }
```

What do you think of moving the above code as well as `responsiveVideos.resize();` into the `doload_llv` function? Then you'd no longer need to add line 41 to 44 because that's exactly what `doload_llv` would call. Does that work? (I might be missing something…)

:)

Reply:
I thought about that as well, but it didn't make sense to me. `responsiveVideos.init()` really only needs to be called once because it only registers window.onload handlers. And putting `responsiveVideos.resize()` into `doload_llv` would call the method twice I think and I am not that much into JS to know if this might be a performance issue.